ajax/libs/redux-search/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "redux-search",
+  "filename": "umd/redux-search.min.js",
+  "version": "2.4.0",
+  "description": "Redux bindings for client-side search",
+  "homepage": "https://bvaughn.github.io/redux-search/",
+  "keywords": [
+    "redux",
+    "search",
+    "filter",
+    "react"
+  ],
+  "author": "Treasure Data",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bvaughn/redux-search.git"
+  },
+  "npmName": "redux-search",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(commonjs|es)/**/*"
```

Comment:
@PeterDaveHello Should we use `dist/umd` as basePath in this case? (There are other folders under `dist`) Compared to https://github.com/cdnjs/cdnjs/pull/12234#discussion_r158249600.

Reply:
@cy91244 As the discussion in https://github.com/cdnjs/cdnjs/pull/12234#discussion_r158249600
Let's update this pattern.

Except this part, everything looks good to me.

Many thanks.

Reply:
@PeterDaveHello Could you please take a look at this question https://github.com/cdnjs/cdnjs/pull/12316#discussion_r158577616? Thanks.